### PR TITLE
added back bqplot to demo @jasongrout

### PR DESCRIPTION
**Pull request by jzf2101**
See #3

### CODE REVIEW

**Comment by jzf2101** [comment]
@ellisonbg this pr now has pyinvoke

File: INSTALL.sh [hunk 1]
```diff
@@ -1,19 +1,30 @@
+#!/usr/bin/env bash
```
Comment (ellisonbg) [inline INSTALL.sh]: Let's move all of the things in the `INSTALL.sh` script to the invoke script as well.

File: tasks.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from invoke import task, run
+
+@task
+def demofiles(ctx, clean=False):
+	if clean:
+		ctx.run('rm -rf demofiles')
+	ctx.run('mkdir demofiles')
```
Comment (ellisonbg) [inline tasks.py]: Let's put the repos in a python list at the top of the file and iterate over them in the function. Makes it easier to extend.

**Comment by jzf2101** [comment]
@ellisonbg I'm on it

File: tasks.py (hunk 2 above)
Comment (jzf2101) [inline tasks.py]: I'm on it

**Comment by jzf2101** [comment]
@ellisonbg PR now passes with the changes you suggeted pls confirm review?

**Pull request merged**